tutorials/algorithms/10_pvqd.ipynb

```diff
@@ -0,0 +1,591 @@
+{
```

Comment:
<p>It might be nice to plot the energies from the <code>exact</code> function you defined previously to show that the exact solution should preserve the energy. With the current version of your code they are not used at all.</p>
<br/>

 _Reply via <a href='https://app.reviewnb.com/Qiskit/qiskit-tutorials/pull/1382/discussion/'>ReviewNB</a>_ <div id='ReviewNBCommentContext-DoNotDelete' style='display:none' data-state='OPEN' data-cellIndex='30'/>